### PR TITLE
Tech debt/update, apply models in house

### PR DESCRIPTION
**Pull request by Maria358**
dev
## JIRA

* [Main JIRA ticket](https://jira.softserve.academy/secure/RapidBoard.jspa?rapidView=id)

## Code reviewers

- [ ] @github_username

### Second Level Review

- [ ] @github_username

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: src/components/AddHouseForm.vue [hunk 1]
```diff
@@ -1,48 +1,54 @@
 <template>
   <form @submit.prevent="addNewHouse" id="house_data_form">
     <p>
-      <label class="dialog-item" for="quantity_flat">Кількість квартир в будинку : </label>
+      <label
+        class="dialog-item"
+        for="
+      flatQuantity"
```
Comment (aleonovdementiev) [inline src/components/AddHouseForm.vue]: Something wrong with indentation here

File: src/components/AddHouseForm.vue [hunk 2]
```diff
@@ -1,48 +1,54 @@
...
+        >Кількість квартир в будинку :
+      </label>
       <InputText
-        id="quantity_flat"
+        id="
+        flatQuantity"
```
Comment (aleonovdementiev) [inline src/components/AddHouseForm.vue]: And here

File: src/store/houses/actions.ts [hunk 3]
```diff
@@ -2,43 +2,50 @@ import { ActionTree } from 'vuex';
...
+import { HouseDTOModel } from './../../shared/models/houseDTO.model';
 
 export const actions: ActionTree<HousesStateInterface, RootStateInterface> & Actions = {
-  [HousesActionsEnum.SET_HOUSES]: ({ commit }) => {
-    HTTP.get(`/cooperations/1/houses`).then((r) => {
-      commit(HousesMutationsEnum.SET_HOUSES, r.data);
-    });
-  },
+  [HousesActionsEnum.SET_HOUSES]: async ({ commit }) => {
+    const { data } = await HTTP.get(`/cooperations/1/houses`);
```
Comment (aleonovdementiev) [inline src/store/houses/actions.ts]: Do you need to hardcode cooperation ID here?

Comment (Maria358) [reply src/store/houses/actions.ts]: It's not my hardcode, but I can correct

Comment (aleonovdementiev) [reply src/store/houses/actions.ts]: From the moment these lines are green, this is your hardcode now))

**Review by aleonovdementiev — APPROVED** [review]

**Review by akononenko2375 — APPROVED** [review]

**Review by rostovskiy88 — APPROVED** [review]

File: src/store/houses/actions.ts (hunk 3 above)
Comment (Maria358) [reply src/store/houses/actions.ts]: I agree) it's my bad

**Pull request merged**